lms/djangoapps/grades/api/tests/test_views.py

```diff
@@ -245,3 +250,228 @@ def test_grade(self, grade, result):
         }
         expected_data.update(result)
         self.assertEqual(resp.data, [expected_data])  # pylint: disable=no-member
+
+
+class GradingPolicyTestMixin(object):
```

Comment:
This looks like it does a really nice job of exercising the view itself (if I understand what the HTTP call here is doing correctly). Are there any tests in the codebase already that explicitly call the `/api/...` endpoint instead of going straight to the view for testing? It would be nice to DDT that/those test(s) with the new URL to make sure it behaves exactly as the old one.

Note: I'm basing this entire comment on the assumption that because the url created in `http_get_for_course` is created with the name of the view, it is calling into that view directly. If that's wrong then disregard!